transcendence-app/.env.test

```diff
@@ -0,0 +1,19 @@
+FORTYTWO_APP_ID=42-intra-app-id
+FORTYTWO_APP_SECRET=42-intra-app-secret
+FORTYTWO_APP_CALLBACK_URL=http://localhost:3000/api/v1/auth/login
+FORTYTWO_APP_AUTHORIZATION_URL=https://api.intra.42.fr/oauth/authorize
+FORTYTWO_APP_TOKEN_URL=https://api.intra.42.fr/oauth/token
+FORTYTWO_APP_PROFILE_URL=https://api.intra.42.fr/v2/me
+NODE_ENV=test
+MEMCACHED_HOST=localhost
+MEMCACHED_PORT=11212
+TRANSCENDENCE_APP_DATA=/tmp
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5433
+POSTGRES_DB=ft_transcendence
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+
+# UUIDs format, generate with uuidgen
+SESSION_SECRET=80fbbf31-7083-419a-8fe1-23f1e10aae80
```

Comment:
I don't get why is .env.test different from .env.sample, in the sense that UUIDs can be hardcoded.

Reply:
Do you mean we could remove the `.env.sample` and have only `.env.test` since we already use it for the e2e tests?

Reply:
discussed on slack

Reply:
We decided to remove the step to set the UUIDs for the `SESSION_SECRET` and the `MEMCACHED_SECRET` from the [installing](https://github.com/42AGV/ft_transcendence#installing) documentation.
For development and testing, these values can be hardcoded for simplicity.

We will have to add this step to the documentation for [deployment](https://github.com/42AGV/ft_transcendence#-deployment-) to a production environment.